<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_Fun</class>
 <widget class="QWidget" name="Frm_Fun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QWidget {
background:white;
}

QPushButton {
color: white;
font:bold 14px;
border:1px solid black;

background: #323232;
}

QLabel {
color: #55555;
background: white;
}*/
*{
font: 13pt;
font-family: &quot;微软雅黑&quot;;
}

QWidget {
color: #c2c2c3;
background: #343639;
}

QLabel {
min-width: 100px;
min-height: 20px;
}

QPushButton[objectName=&quot;btnTeach&quot;] {
border: no;
/*outline: no;*/
min-width: 100px;
min-height: 100px;
image: url(&quot;:/Res/image/function_teach.png&quot;); 
}

QPushButton[objectName=&quot;btnTeach&quot;]:pressed {
border: no;
min-width: 100px;
min-height: 100px;
/*
padding-top: -2px;
padding-bottom: -2px;*/
padding: 2px 2px 2px 2px;
image: url(&quot;:/Res/image/function_teach.png&quot;); 
}

QPushButton[objectName=&quot;btnRun&quot;] {
border: no;
/*outline: no;*/
min-width: 100px;
min-height: 100px;
image: url(&quot;:/Res/image/function_run.png&quot;); 
}

QPushButton[objectName=&quot;btnRun&quot;]:pressed {
border: no;
min-width: 100px;
min-height: 100px;
/*
padding-top: -2px;
padding-bottom: -2px;*/
padding: 2px 2px 2px 2px;
image: url(&quot;:/Res/image/function_run.png&quot;); 
}

QPushButton[objectName=&quot;btnSysS&quot;] {
border: no;
/*outline: no;*/
min-width: 100px;
min-height: 100px;
image: url(&quot;:/Res/image/function_setup.png&quot;); 
}

QPushButton[objectName=&quot;btnSysS&quot;]:pressed {
border: no;
min-width: 100px;
min-height: 100px;
/*
padding-top: -2px;
padding-bottom: -2px;*/
padding: 2px 2px 2px 2px;
image: url(&quot;:/Res/image/function_setup.png&quot;); 
}

QPushButton[objectName=&quot;btnStkS&quot;] {
border: no;
/*outline: no;*/
min-width: 100px;
min-height: 100px;
image: url(&quot;:/Res/image/function_stack.png&quot;); 
}

QPushButton[objectName=&quot;btnStkS&quot;]:pressed {
border: no;
min-width: 100px;
min-height: 100px;
/*
padding-top: -2px;
padding-bottom: -2px;*/
padding: 2px 2px 2px 2px;
image: url(&quot;:/Res/image/function_stack.png&quot;); 
}

QPushButton[objectName=&quot;btnTimS&quot;] {
border: no;
/*outline: no;*/
min-width: 100px;
min-height: 100px;
image: url(&quot;:/Res/image/function_time.png&quot;); 
}

QPushButton[objectName=&quot;btnTimS&quot;]:pressed {
border: no;
min-width: 100px;
min-height: 100px;
/*
padding-top: -2px;
padding-bottom: -2px;*/
padding: 2px 2px 2px 2px;
image: url(&quot;:/Res/image/function_time.png&quot;); 
}

QPushButton[objectName=&quot;btnMonitor&quot;] {
border: no;
/*outline: no;*/
min-width: 100px;
min-height: 100px;
image: url(&quot;:/Res/image/function_monitor.png&quot;); 
}

QPushButton[objectName=&quot;btnMonitor&quot;]:pressed {
border: no;
min-width: 100px;
min-height: 100px;
/*
padding-top: -2px;
padding-bottom: -2px;*/
padding: 2px 2px 2px 2px;
image: url(&quot;:/Res/image/function_monitor.png&quot;); 
}

QPushButton[objectName=&quot;btnLog&quot;] {
border: no;
/*outline: no;*/
min-width: 100px;
min-height: 100px;
image: url(&quot;:/Res/image/function_note.png&quot;); 
}

QPushButton[objectName=&quot;btnLog&quot;]:pressed {
border: no;
min-width: 100px;
min-height: 100px;
/*
padding-top: -2px;
padding-bottom: -2px;*/
padding: 2px 2px 2px 2px;
image: url(&quot;:/Res/image/function_note.png&quot;); 
}
QPushButton[objectName=&quot;btnAuthority&quot;] {
border: no;
/*outline: no;*/
min-width: 100px;
min-height: 100px;
image: url(&quot;:/Res/image/function_authority.png&quot;); 
}

QPushButton[objectName=&quot;btnAuthority&quot;]:pressed {
border: no;
min-width: 100px;
min-height: 100px;
/*
padding-top: -2px;
padding-bottom: -2px;*/
padding: 2px 2px 2px 2px;
image: url(&quot;:/Res/image/function_authority.png&quot;); 
}

QPushButton[objectName=&quot;btnHelp&quot;] {
border: no;
/*outline: no;*/
min-width: 100px;
min-height: 100px;
image: url(&quot;:/Res/image/function_help.png&quot;); 
}

QPushButton[objectName=&quot;btnHelp&quot;]:pressed {
border: no;
min-width: 100px;
min-height: 100px;
/*
padding-top: -2px;
padding-bottom: -2px;*/
padding: 2px 2px 2px 2px;
image: url(&quot;:/Res/image/function_help.png&quot;); 
}

QPushButton[objectName=&quot;btnManufacturer&quot;] {
border: no;
/*outline: no;*/
min-width: 101px;
min-height: 101px;
image: url(&quot;:/Res/image/function_vendor.png&quot;); 
}

QPushButton[objectName=&quot;btnManufacturer&quot;]:pressed {
border: no;
min-width: 101px;
min-height: 101px;
/*
padding-top: -2px;
padding-bottom: -2px;*/
padding: 2px 2px 2px 2px;
image: url(&quot;:/Res/image/function_vendor.png&quot;); 
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>90</number>
   </property>
   <property name="topMargin">
    <number>100</number>
   </property>
   <property name="rightMargin">
    <number>90</number>
   </property>
   <property name="bottomMargin">
    <number>100</number>
   </property>
   <property name="horizontalSpacing">
    <number>25</number>
   </property>
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnTeach">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>示教</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>运行</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnStkS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>堆叠设置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnMonitor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>监视</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>记录查询</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSysS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>设置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnTimS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>时间设置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>帮助</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnAuthority">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>用户管理</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="4">
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QPushButton" name="btnManufacturer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>厂商维护</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
